data-raw/organizations/TRANSACCESS/prepare-TRANSACCESS.R

```diff
@@ -0,0 +1,57 @@
+# TRANSACCESS Preparation Script
+
+# The Transaccess dataset by Sommerer, Tallberg, and Squatrito
+# contains information on the access of transnational actors to IGO bodies.
+
+# This is a template for importing, cleaning, and exporting data
+# ready for many packages universe.
+
+# Stage one: Collecting data
+TRANSACCESS <- haven::read_dta("data-raw/organizations/TRANSACCESS/Transaccess data.dta")
+
+# Stage two: Correcting data
+# In this stage you will want to correct the variable names and
+# formats of the 'TRANSACCESS' object until the object created
+# below (in stage three) passes all the tests.
+# We recommend that you avoid using one letter variable names to keep
+# away from issues with ambiguous names down the road.
+TRANSACCESS <- as_tibble(TRANSACCESS) %>%
+  manydata::transmutate(igoID = IO,
+                        Label = manypkgs::standardise_titles(IOname),
+                        igobodyID = IDIOBO,
+                        IGOBody = manypkgs::standardise_titles(IObodyname)) %>%
```

Comment:
Should these be another database? i.e. 'bodies' as distinct from 'organizations'?